components/service/glean/build.gradle

```diff
@@ -16,6 +16,12 @@ apply plugin: 'kotlin-android'
  */
 String GLEAN_PING_SCHEMA_GIT_HASH = "8939bcb"
 String GLEAN_PING_SCHEMA_URL = "https://raw.githubusercontent.com/mozilla-services/mozilla-pipeline-schemas/$GLEAN_PING_SCHEMA_GIT_HASH/schemas/glean/baseline/baseline.1.schema.json"
+// NOTE: this is also defined in scripts/sdk_generator.gradle
+String MINICONDA_VERSION = "4.5.11"
```

Comment:
Is there any way to share this instead of declaring it twice?

Reply:
Not that I was able to find -- Groovy/Gradle is weird -- but I'll look at it again today with fresh eyes.

Reply:
I think I found a way forward.  The `sdk_generator.gradle` file can specify the buildConfig variable used to find the Python installation during testing (to validate against schemas).  Then we don't have to do it from glean's `build.gradle`.

Reply:
Ah, interesting. I wonder how that would look in code, but sounds like a nice alternative to the current state!

Reply:
This PR has been updated with those changes.  Let me know what you think.